src/library/entities/Library.java

```diff
@@ -84,153 +86,155 @@ public static synchronized void SaVe() {
     }
 
     
-    private long gEt_NeXt_ItEm_Id() {
-        return NeXt_ItEm_Id++;
+    private long getNextItemId() {
+        return nextItemId++;
     }
 
     
-    private long gEt_NeXt_PaTrOn_Id() {
-        return NeXt_PaTrOn_Id++;
+    private long getNextPatronId() {
+        return nextPatronId++;
     }
 
     
-    private long gEt_NeXt_LoAn_Id() {
-        return NeXt_lOaN_Id++;
+    private long getNextLoanId() {
+        return nextLoanId++;
     }
 
     
-    public List<Patron> lIsT_PaTrOnS() {        
-        return new ArrayList<Patron>(PaTrOnS.values()); 
+    public List<Patron> listPatrons() {
+        return new ArrayList<Patron>(patrons.values());

```

Comment:
Not sure if constructors cound as methods in this instance?

> The argument list of any method calls MUST NOT contain any other method calls. E.g.

Reply:
Not sure. @haynescrossley

Reply:
Nothing specifically listed in CSG about constructors. Though I don't think there is any issue here. Its not a method, calling another method. Just returning the hash map values.

CSG example of what NOT to do:

someOtherVar = someOtherMethod(someMethod(x, y, z), p, q, r);